.rultor.yml

```diff
@@ -2,6 +2,8 @@ install: |-
   sudo apt-get install bsdmainutils
   sudo gem install pdd
   sudo gem install est
+docker:
+  image: mostalive/ubuntu-14.04-oracle-jdk8
```

Comment:
@longtimeago please surround it with quote as mentioned in the issue, it should look like `image: "mostalive/ubuntu-14.04-oracle-jdk8"`